.gitignore

```diff
@@ -0,0 +1,3 @@
+/.idea/
+/node_modules/
+/docker-compose.override.yml
```

Comment:
Can we ignore different OS generated files here?

Reply:
Personally, I would prefer OS generated files to be gitignored at the global level by devs by adding the files for their OS to their account's gitignore file, but we can add them here easy enough.

Can you provide a list of files you'd like to ignore?

Reply:
I looked at repos like react, view, electron, and node and they all have `.DS_Store`, but no windows thumbs file which I would have suggested. We probably just need the ds_store file then

Also, I wonder if we should add support for other IDEs here since others can technically contribute. Thoughts? eg: visual studio, sublime, etc. https://github.com/facebook/react/blob/master/.gitignore

Reply:
Added in `.DS_Store`. We can start there and see if we need to add more later.